src/Examples/Examples.GameOfLife/Examples.GameOfLife.fsproj

```diff
@@ -22,6 +22,7 @@
 
     <ItemGroup>
         <PackageReference Include="Avalonia.Desktop" Version="0.9.2" />
+        <PackageReference Update="FSharp.Core" Version="4.6.2" />
```

Comment:
Could you explain why you added this here (and elsewhere) ?

Reply:
Without this reference update sample application failed to start on OSX with this error:

```
Could not load file or assembly 'FSharp.Core, Version=4.6.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The located assembly's manifest definition does not match the assembly reference. (0x80131040)
```

I've changed the version of library to 4.7.0. This doesn't produce a warning during the build on Windows, and still fixes issue with running on OSX.

Reply:
Strange, I'm also running on MacOS without problems. It's ok.